src/main/kotlin/com/ajaxproject/telegrambot/bot/BankInfoBot.kt

```diff
@@ -0,0 +1,40 @@
+package com.ajaxproject.telegrambot.bot
+
+import com.ajaxproject.telegrambot.bot.commands.AbstractCommand
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.telegram.telegrambots.bots.TelegramLongPollingBot
+import org.telegram.telegrambots.meta.api.objects.Update
+
+@Component
+class BankInfoBot(
+    @Qualifier("commandsMap")
+    private val commandsMapping: Map<String, AbstractCommand>,
+) : TelegramLongPollingBot() {
+
+    @Value("\${credentials.botUsername}")
+    private val botUsername: String = ""
+
+    @Value("\${credentials.botToken}")
+    private val token: String = ""
+
+    override fun getBotUsername(): String = botUsername
+    override fun getBotToken(): String = token
+
+    override fun onUpdateReceived(update: Update?) {
```

Comment:
I believe an update can't be null, because what is the reason for sending such an update?) so, you can make type not-nullable here and simplify code in conditions